src/utils/index.ts

```diff
@@ -60,6 +76,21 @@ const ETHERSCAN_PREFIXES: { [chainId in ChainId]: string } = {
   296: CHAINS[ChainId.HEDERA_TESTNET].blockExplorerUrls?.[0] || '',
   329847900: CHAINS[ChainId.NEAR_MAINNET].blockExplorerUrls?.[0] || '',
   329847901: CHAINS[ChainId.NEAR_TESTNET].blockExplorerUrls?.[0] || '',
+  1: '',
+  137: '',
+  66: '',
+  106: '',
+  56: '',
+  42220: '',
```

Comment:
in this mapping, can we use [ChainId.ETHEREUM] instead of direct number mapping .... like we are doing for every mapping

Reply:
fixed